libs/core/enums/animeType.ts

```diff
@@ -0,0 +1,9 @@
+/** Anime type enum. */
+export enum AnimeType {
+  TV = 'TV',
+  OVA = 'OVA',
+  MOVIE = 'MOVIE',
+  SPECIAL = 'SPECIAL',
+  ONA = 'ONA',
+  MUSIC = 'MUSIC',
+}
```

Comment:
Use `CamelCase` for enums
```suggestion
/** Anime type enum. */
export enum AnimeType {
  TV = 'TV',
  OVA = 'OVA',
  Movie = 'MOVIE',
  Special = 'SPECIAL',
  ONA = 'ONA',
  Music = 'MUSIC',
}
```

Reply:
Corrected